src/VolumeDrawable.ts

```diff
@@ -662,62 +667,28 @@ export default class VolumeDrawable {
 
     // create new
     if (isPathtrace) {
-      this.pathTracedVolume = new PathTracedVolume(this.volume);
+      this.pathTracedVolume = new PathTracedVolume(this.volume, this.settings);
       this.volumeRendering = this.pathTracedVolume;
       this.volumeRendering.setRenderUpdateListener(this.renderUpdateListener);
     } else {
       // Quietly swap the RayMarchedAtlasVolume for a 2D slice renderer when our render mode
       // is XY/Z mode.
-      if (this.viewMode === "XY" || this.viewMode === "Z") {
-        this.atlas2DSlice = new Atlas2DSlice(this.volume);
+      if (this.viewMode === Axis.Z) {
+        this.atlas2DSlice = new Atlas2DSlice(this.volume, this.settings);
         this.volumeRendering = this.atlas2DSlice;
       } else {
-        this.rayMarchedAtlasVolume = new RayMarchedAtlasVolume(this.volume);
+        this.rayMarchedAtlasVolume = new RayMarchedAtlasVolume(this.volume, this.settings);
         this.volumeRendering = this.rayMarchedAtlasVolume;
       }
 
-      for (let i = 0; i < this.volume.num_channels; ++i) {
-        if (this.volume.getChannel(i).loaded) {
-          this.volumeRendering.onChannelData([i]);
-        }
-      }
       if (this.renderUpdateListener) {
         this.renderUpdateListener(0);
       }
     }
 
     // ensure transforms on new volume representation are up to date
-    this.volumeRendering.setTranslation(this.translation);
-    this.volumeRendering.setRotation(this.rotation);
-
     this.PT = isPathtrace;
 
-    this.setChannelAsMask(this.maskChannelIndex);
-    this.setMaskAlpha(this.maskAlpha);
-    this.setScale(this.volume.scale);
-    this.setBrightness(this.getBrightness());
-    this.setDensity(this.getDensity());
-    this.setGamma(this.gammaMin, this.gammaLevel, this.gammaMax);
-    this.setFlipAxes(this.flipX, this.flipY, this.flipZ);
-    this.setZSlice(this.zSlice);
-
-    // reset clip bounds
-    this.setAxisClip("x", this.bounds.bmin.x, this.bounds.bmax.x);
-    this.setAxisClip("y", this.bounds.bmin.y, this.bounds.bmax.y);
-    this.setAxisClip("z", this.bounds.bmin.z, this.bounds.bmax.z);
-    this.updateClipRegion(
-      this.bounds.bmin.x + 0.5,
-      this.bounds.bmax.x + 0.5,
-      this.bounds.bmin.y + 0.5,
-      this.bounds.bmax.y + 0.5,
-      this.bounds.bmin.z + 0.5,
-      this.bounds.bmax.z + 0.5
-    );
-    this.setRayStepSizes(this.primaryRayStepSize, this.secondaryRayStepSize);
-
-    this.setShowBoundingBox(this.showBoundingBox);
-    this.setBoundingBoxColor(this.boundingBoxColor);
-
```

Comment:
should we be calling updateSettings here or is the one passed in to the constructors the way it happens?